Public/Src/Cache/ContentStore/Grpc/ContentStore.proto

```diff
@@ -199,6 +201,19 @@ message ExistenceResponse {
     ResponseHeader Header = 1;
 }
 
+// Delete
+message DeleteContentRequest {
+    string traceId = 1;
+    int32 hashType = 2;
+    bytes contentHash = 3;
+}
+
+message DeleteContentResponse {
+    ResponseHeader Header = 1;

```

Comment:
>H [](start = 19, length = 1)

casing is off.
Protobuf recommended casing is camlCase. The C# codegen will make it PascalCase....

I see you just copied, but you should probably do a pass before too many folks take dependencies and clean it up.

Reply:
I hope that isn't a dangerous change! I'll migrate this entire file to camelCase after this PR is checked in.

---
In reply to: [303659836](https://github.com/microsoft/BuildXL/pull/599#discussion_r303659836) [](ancestors = 303659836)

Reply:
I don't see a point in changing this. It's purely cosmetic and more likely to cause problems than it is to help.

---
In reply to: [303673571](https://github.com/microsoft/BuildXL/pull/599#discussion_r303673571) [](ancestors = 303673571,303659836)